### ISSUE TITLE
Check a group of input (well a group of 2)...

### ISSUE THREAD

**Issue by djournois** [PascaleBeier/bootstrap-validate#131]
Do you imagine a way to  1) check that emails syntaxes are ok 2) have at least one of the two address filled ?

    <div class="form-group">
        <label for="emailpro" class="control-label"><strong>Pro email</strong></label>
        <input type="text" class="form-control" id="emailpro" name="emailpro">
    </div>
    <script>
        bootstrapValidate('#emailpro', 'email:does not look like an email address');
    </script>

    <div class="form-group">
        <label for="emailperso"><strong>Perso email</strong></label>
        <input type="text" class="form-control" id="emailperso" name="emailperso">
    </div>
    <script>
        bootstrapValidate('#emailperso', 'email:does not look like an email address');
    </script>

**Comment by PascaleBeier** [comment]
I dont think this works right now, this would be something like a "required unless" another field is present rule.

**Comment by PascaleBeier** [comment]
Added in v2.3.0: requiredUnless supports the 'at least one of these fields' use case while keeping each field's own validation rules separate. Release: https://github.com/PascaleBeier/bootstrap-validate/releases/tag/v2.3.0 Docs: https://bootstrap-validate.js.org/v2/rules/requiredUnless/
